app/models/database_backup.rb

```diff
@@ -1,13 +1,8 @@
 class DatabaseBackup < ApplicationRecord
-  SUPPORTED_DEPOTS = {
-    'smb'   => 'Samba',
-    'nfs'   => 'Network File System',
-    's3'    => 'AWS S3',
-    'swift' => 'OpenStack Swift'
-  }.freeze
+  SUPPORTED_DEPOTS = %w[FileDepotSmb FileDepotNfs FileDepotS3 FileDepotSwift].freeze
 
   def self.supported_depots
-    SUPPORTED_DEPOTS
+    SUPPORTED_DEPOTS.map { |model| [model, model.constantize.display_name] }.to_h
```

Comment:
Since this is a class method, should we cache this?  Otherwise we are re-building this object ever this everytime this method is called.

Reply:
display_name can change based on the localization, so we can't cache it.

Reply:
Bah... you are right, good point.  Your comment below does give pause, but I was looking at the `.to_h` and thought that it was just replicating the deleted hash from above (which it is not).

I basically then agree with @Fryguy pending the https://github.com/ManageIQ/manageiq/pull/20941#discussion_r574803529 below.